virtual_labs/tests/utils.py

```diff
@@ -243,3 +245,18 @@ async def create_confirmed_setup_intent(customer_id: str) -> SetupIntent:
     )
 
     return intent
+
+
+def wait_until(
+    somepredicate: typing.Any,
```

Comment:
I would type this as `typing.Callable[..., bool]`
See: https://docs.python.org/3/library/typing.html#annotating-callable-objects

But this is pretty minor, feel free to ignore.

Reply:
done